specs/simple-serialize.md

```diff
@@ -449,6 +449,77 @@ assert item_index == start + LENGTH_BYTES + length
 return typ(**values), item_index
 ```
 
+### Tree hash
+
```

Comment:
Need to be clearer about defining `tree_hash` as a function separate from `merkle_hash`. I think just stating it as `tree_hash` this header will help.

Reply:
Done!